src/Raven.Server/ServerWide/RawDatabaseRecord.cs

```diff
@@ -238,7 +238,6 @@ public RawDatabaseRecord GetShardedDatabaseRecord(int shardNumber)
             sharding.Modifications = new DynamicJsonValue(sharding)
             {
                 [nameof(ShardingConfiguration.BucketRanges)] = null,
```

Comment:
I think we need to remove the `BucketRanges` here, otherwise this will be striped and we wouldn't able to tell the correct shard?

Reply:
I don't think we need the `BucketRanges`, we only use `ShardedDocumentDatabase.ShardingConfiguration` for updating `BucketAndEtag` indexes and `BucketStats` tree.
so we don't care about shard number, just the bucket number - and `Prefixed` is enough for that